### ISSUE TITLE
[9.3.*] Tasks set to Done doesn't show on the planning list view

### ISSUE THREAD

**Issue by M-Falken** [glpi-project/glpi#4616]
Hi,
I don't think this is a normal behaviour
In any cases the Tasks must always appear on the planning whatever the status are.
But Tasks passed to the "Done" status do not appear on the schedule list view.

Steps  #to reproduce:

1. Open a ticket
2. Schedule a task with status "To do"
3. Look at the dynamic schedule, the task is present
4. Look at the schedule list view, the task is present
5. Go to the ticket processing tab
6. Check the task box to switch to status "Done"
7. Look at the dynamic schedule, the task is present
8. Look at the list view of the schedule, **no task appears**.

So basically a tech can set a task to "Done" without any solution and claim that he didn't see the task on his own list view.

I suppose this is not a voluntary mode of operation of this display.
And may be related to the behaviour of https://github.com/glpi-project/glpi/issues/4313
```
9.3 prod
9.3.1  #test
```

**Comment by M-Falken** [comment]
Tested today with the latest nightly. No changes.
The task must always appear on the reception schedule even if a solution has been filled in as long as the manager has not changed the status of the task to "Done".
Otherwise the task disappears from the schedule of the reception when the solution has not been validated.
Thanks
